redis/src/types.rs

```diff
@@ -1351,6 +1351,68 @@ pub trait RedisWrite {
     /// Appends an empty argument to the command, and returns a
     /// [`std::io::Write`] instance that can write to it.
     fn writer_for_next_arg(&mut self) -> impl std::io::Write + '_;
+
+    #[cfg(feature = "bytes")]
+    /// Appends an empty argument to the command, and returns a
```

Comment:
This would be the final part of the PR, but the documentation should include the safety guarantees / assumptions of this function - for example, what happens if the user writes more than a single argument, or doesn't write any data at all.

Reply:
When adding that I should also add those same safety guarantees to `writer_for_next_arg`

Reply:
I've added some documentation to `bufmut_for_next_arg` and `writer_for_next_arg` about possible issues.